src/app/error/components/500.tsx

```diff
@@ -54,28 +24,49 @@ const Error500 = () => {
           textAlign: "center",
         }}
       >
-        <BoxWrapper>
-          <Typography variant="h1">500</Typography>
+        <Styled.BoxWrapper>
           <Typography
-            variant="h5"
-            sx={{ mb: 1, fontSize: "1.5rem !important" }}
+            variant="h1"
+            sx={{
+              mb: 2.5,
+              fontSize: { md: "2rem !important", xs: "1.5rem !important" },
+            }}
           >
-            Internal server error 👨🏻‍💻
+            Error del servidor 👨🏻‍💻
+          </Typography>
+          <Typography variant="body1">
+            ¡Lo sentimos, ha ocurrido un error interno en el servidor! Estamos
+            trabajando para solucionarlo lo antes posible. Gracias por tu
+            paciencia.
```

Comment:
For this kind of errors, I like to give the users two ways of solving this:
- Try again later
- I that does not work, contact us.

So, I suggest this message:
> Lo sentimos, no hemos podido procesar tu petición correctamente. Por favor, vuelve a intentarlo más tarde
> Si el problema persiste, contáctanos.

And then add a contact button.